tools/virt-builder/about.md

```diff
@@ -47,38 +47,45 @@ root password, only the bare minimum installed software, etc.).
 
 Import the above disk image into libvirt:
 
-    $ virt-install --import \
-       --name f23vm1 --ram 2048 \
-       --disk path=fedora-23.img,format=raw --os-variant fedora23
+    $ virt-install \
+       --name f27vm1 --ram 2048 \
+       --disk path=fedora-27.img,format=raw \
+       --os-variant fedora26 \
+       --import \
+
+Note: The `--os-variant` doesn't _strictly_ have to be F27; the nearest
+possible available Fedora variant is fine.  To find the list of variants
+for your current Fedora release, run: `osinfo-query os | grep fedora`.
 
 ## Examples to create custom virtual machines
 
-###  Create a Fedora 23 VM and update it
+###  Create an up-to-date Fedora 27 VM
 
-Prepare a QCOW2 Fedora 23 VM, with 40GB disk size, and update to latest
+Prepare a QCOW2 Fedora 27 VM, with 40GB disk size, and update to latest
 available packages:
 
-    $ virt-builder fedora-23 -o f23vm2.qcow2 --format qcow2 \
+    $ virt-builder fedora-27 -o f27vm2.qcow2 --format qcow2 \
         --update --selinux-relabel --size 40G
 
 Import the disk image into libvirt, and provide it 4GB of memory:
 
-    $ virt-install --name f23vm2 --ram 4096 \
-        --disk path=f23vm2.qcow2,format=qcow2,cache=writeback \
-        --nographics --import --os-variant fedora23
+    $ virt-install --name f27vm2 --ram 4096 \
+        --disk path=f27vm2.qcow2,format=qcow2,cache=writeback \
+        --nographics --os-variant fedora26 --import
 
-### Create a rawhide VM
+### Create a rawhide VM disk image
 
-This creates a rawhide VM:
+This creates a rawhide disk :
 
-    $ virt-builder fedora-23 \
+    $ virt-builder fedora-27 \
         --install fedora-repos-rawhide \
         --edit '/etc/yum.repos.d/fedora-rawhide.repo: s/enabled=0/enabled=1/' \
         --update \
         --selinux-relabel --size 40G \
         --output rawhide-vm.img
 
-Import it into libvirt, with similar sytax as shown in previous examples.
+Import it into libvirt with `virt-install` (as shown in previous
```

Comment:
I'm still somehow a bit against the wrapping here.

Reply:
It's the 72 character line-wrap again.